### DISCUSSION TITLE
where exactly do you report hosted malware used in phishing campaigns to github?

### DISCUSSION THREAD

**Discussion by GHReport1** [community/community#125583] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

it's a circle jerk of 'click here' articles.  I'm pretty sure the repository maintainers are not innocent -- so, i'm not going to file a issue there.

https://github[.]com/ustaxes/UsTaxes/files/15419438/2023TaxDocuments.zip

password received in the phishing email to extract is '2030'.

**Comment by mickhence** [comment]
If you've stumbled upon hosted malware being used in phishing campaigns on GitHub, it's crucial to report it promptly to GitHub's Security team to ensure swift action is taken to mitigate the threat. Here's how you can do it:

Identify the Repository: Note down the URL of the repository containing the malware. Make sure to include any relevant information about the specific files or code segments involved.

Report to GitHub Security: Visit GitHub's Security Advisories page (https://github.com/contact/security) to report security vulnerabilities and incidents. Choose the appropriate category, likely "Report a vulnerability in GitHub-hosted repositories," and provide all necessary details, including the repository URL, description of the issue, and any supporting evidence or context.

Provide Details: In your report, be as detailed as possible about the nature of the malware, its potential impact, and any actions you've taken to mitigate its spread or damage.

Stay Engaged: After submitting your report, stay engaged with GitHub's Security team if they require further information or updates regarding the incident. Prompt and thorough communication can expedite the resolution process.

By reporting hosted malware used in phishing campaigns to GitHub, you're not only helping to protect GitHub's users and infrastructure but also contributing to the broader effort of combating cyber threats across the internet.

**Reply by ebndev** [reply]
Hi @mickhence, thanks for wanting to participate in GitHub Discussions! 

Please note: We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ebndev** [comment]
Hi @GHReport1, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by GHReport1** [reply]
As i mentioned, the reporting tool and system is just a circle jerk of useless links. I got the 'file download' link in a phishing e-mail one of my users received. And, I found it interesting enough that they were hiding it behind what looks to be a legitimate github repository and/or they're just planting seeds to make it look like it's a legitimate/active repository. But, of course, I have my suspicions about the maintainers of that repository (how they didn't notice the file was specifically uploading to one of their repository folders that they created).

It isn't a user, issue, pull request, comment or anything that has a 'report' link I can use. Maybe 'ai' needs to do a better job parsing my comment .... It is a malicious download link intentionally stored on a active repository that was sent in a phishing e-mail to my end user.

**Comment by GHReport1** [comment]
As i mentioned, the reporting tool and system is just a circle jerk of useless links.  I got the 'file download' link in a phishing e-mail one of my users received.  And, I found it interesting enough that they were hiding it behind what looks to be a legitimate github repository and/or they're just planting seeds to make it look like it's a legitimate/active repository.  But, of course, I have my suspicions about the maintainers of that repository (how they didn't notice the file was specifically uploading to one of their repository folders that they created).

It isn't a user, issue, pull request, comment or anything that has a 'report' link I can use.  Maybe 'ai' needs to do a better job parsing my comment .... It is a malicious download link intentionally stored on a active repository that was sent in a phishing e-mail to my end user.
